pages/solid-events.md

```diff
@@ -20,6 +20,8 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 - Have a topic suggestion for an upcoming Solid World? You can fill out the [Solid World Topic Suggestions form](https://es1cz4pb7oi.typeform.com/to/yQNMvK0N).
 
+- Have another Solid oriented event to share? Make a pull request to the [solidproject.org events page](https://github.com/solid/solidproject.org/blob/main/pages/solid-events.md).
```

Comment:
Good idea - might be easier to link directly to the "edit" UI though, rather than have people hunt for the pencil icon or figure out how to fork a Git repo themselves :)

```suggestion
- Have another Solid oriented event to share? Make a pull request to the [solidproject.org events page](https://github.com/solid/solidproject.org/edit/main/pages/solid-events.md).
```